packages/jss/src/index.d.ts

```diff
@@ -15,13 +15,17 @@ type NormalCssValues<K> = K extends keyof NormalCssProperties
   ? NormalCssProperties[K] | JssValue
   : JssValue
 
-export type JssStyle = {
-  [K in keyof NormalCssProperties | string]:
-    | NormalCssValues<K>
-    | JssStyle
-    | Func<NormalCssValues<K> | JssStyle | undefined>
-    | Observable<NormalCssValues<K> | JssStyle | undefined>
-}
+export type JssStyle =
+  | {
+      [K in keyof NormalCssProperties]:
+        | NormalCssValues<K>
+        | JssStyle
+        | Func<NormalCssValues<K> | JssStyle | undefined>
+        | Observable<NormalCssValues<K> | JssStyle | undefined>
+    }
+  | {
+      [K: string]: JssStyle | Func<JssStyle> | Observable<JssStyle | undefined>
```

Comment:
why is this value different than the first value, isn't it supposed to be the exact saame thing?

```
        | NormalCssValues<K>
        | JssStyle
        | Func<NormalCssValues<K> | JssStyle | undefined>
        | Observable<NormalCssValues<K> | JssStyle | undefined>
```

Reply:
Actually no, because the string is not a key of NormalCssValues.

Reply:
I just removed NormalCssValues

Reply:
NormalCssValues has no effect there

Reply:
But I missed jssValue